addons/account/tests/common.py

```diff
@@ -194,32 +171,84 @@ def change_company_country(cls, company, country):
+        # OVERRIDE
+        company = super().setup_independent_company(**kwargs)
+        if not company:
+            company = cls._create_company(name='company_1_data', **kwargs)
+        return company
+
+    @classmethod
+    def _create_company(cls, **create_values):
+        country_code = getattr(cls, 'country_code', None)
+        if country_code:
+            country = cls.env['res.country'].search([('code', '=', country_code.upper())])
+            if not country:
+                raise ValueError('Invalid country code')
+            create_values.update({
+                'country_id': create_values.get('country_id', country.id),
+                'currency_id': create_values.get('currency_id', country.currency_id.id),
+            })
+        chart_template_ref = getattr(cls, 'chart_template', None)
+        company = super()._create_company(**create_values)
+        cls._use_chart_template(company, chart_template_ref)
+        if create_values.get('currency_id'):
+            company.currency_id = create_values['currency_id']
+        return company
+
+    @classmethod
+    def _create_product(cls, **create_values):
+        # OVERRIDE
+        create_values.setdefault('property_account_income_id', cls.company_data['default_account_revenue'].id)
+        create_values.setdefault('property_account_expense_id', cls.company_data['default_account_expense'].id)
+        create_values.setdefault('taxes_id', [Command.set(cls.tax_sale_a.ids)])
+        return super()._create_product(**create_values)
+
+    @classmethod
+    def get_default_groups(cls):
+        groups = super().get_default_groups()
+        return groups | cls.env.ref('account.group_account_manager') | cls.env.ref('account.group_account_user')
+
+    @classmethod
+    def _create_user(cls, groups, **create_values):
+        return super()._create_user(
+            groups,
+            **{
+                'name': 'Because I am accountman!',
+                'login': 'accountman',
+                'password': 'accountman',
+                'email': 'accountman@test.com',
+                **create_values,
+            },
+        )
+
+    @classmethod
+    def setup_other_currency(cls, code, **kwargs):
+        if 'rates' not in kwargs:
+            return super().setup_other_currency(code, rates=[('2016-01-01', 3.0), ('2017-01-01', 2.0)], **kwargs)
+        return super().setup_other_currency(code, **kwargs)
+
+    @classmethod
+    def _use_chart_template(cls, company, chart_template_ref=None):
+        chart_template_ref = chart_template_ref or cls.env['account.chart.template']._guess_chart_template(company.country_id)
+        template_vals = cls.env['account.chart.template']._get_chart_template_mapping()[chart_template_ref]
+        template_module = cls.env['ir.module.module']._get(template_vals['module'])
+        if template_module.state != 'installed':
+            raise SkipTest(f"Module required for the test is not installed ({template_module.name})")
```

Comment:
Note: It could be interested to check which tests are run during the standalone tests on runbot to ensure all localization builds are not also running all generic tests

Reply:
So, only every test that has the tag `post_install_l10n`. 
Currently, 8 occurences in test files outside of l10n modules